dynamic-plugins/wrappers/roadiehq-backstage-plugin-argo-cd-backend-dynamic/package.json

```diff
@@ -29,7 +29,7 @@
     "@backstage/backend-common": "^0.21.6",
     "@backstage/backend-plugin-api": "^0.6.12",
     "@backstage/backend-dynamic-feature-service": "^0.2.2",
```

Comment:
I guess we don't need this dependency anymore (not related to this change BTW).

Reply:
you can use the `suggestion` feature to suggest code changes. I'm nor sure if you mean to just remove this line
```
"@backstage/backend-dynamic-feature-service": "^0.2.2",
```

or something else / more widespread? or suggesting another PR?